phylogenetic/defaults/clade-i/config.yaml

```diff
@@ -19,14 +19,14 @@ auspice_name: "mpox_clade-I"
 filter:
   min_date: 1900
   min_length: 100000
-  exclude_where: 'clade!=I'
```

Comment:
Sorry, I'm still confused...

After reading https://github.com/nextstrain/mpox/issues/283#issuecomment-2448388791, I was under the impression that the change would be to replace `subsample.<sample>.exclude_where: 'clade!=I'` with `subsample.<sample>.query: clade in I, Ia, Ib`. But now I see that it's replacing `filter.exclude_where: 'clade!=I'` with `subsample.<sample>.query: clade in I, Ia, Ib`. Why not `filter.query` to keep consistent with having all filters in `filter`?

Reply:
Good question! 

This is my interpretation of the workflow based on the other mpox build configs (e.g. [hmpxv_big](https://github.com/nextstrain/mpox/blob/4ce0b5bdc6a0c21d1eaee95ff7d17725b1470b27/phylogenetic/defaults/hmpxv1_big/config.yaml#L19-L36)):

1. Use `filter` to filter for "good" sequences (by date, length, QC)
2. Use `subsample` to filter and subsample by clade and lineages

The clade I build actually deviated from this pattern by [adding `filter.exclude_where`](https://github.com/nextstrain/mpox/commit/1140955fae354b87b3c92e2d5a8191898a6f1312#diff-041205ab957fdd3aac682dffdc4b564dba6bbfd077747724676d6a60f46887ad). Using `subsample.<sample>.query` here to bring it back to the pattern used by other builds.